frontend/src/pages/edit-driver-profile/components/edit-avatar-form/edit-avatar-form.tsx

```diff
@@ -0,0 +1,94 @@
+import { Button, ImageSelector } from '~/libs/components/components.js';
+import { ImgPath } from '~/libs/enums/enums.js';
+import { getValidClassNames } from '~/libs/helpers/helpers.js';
+import {
+  useAppDispatch,
+  useAppForm,
+  useCallback,
+  useMemo,
+  useRef,
+} from '~/libs/hooks/hooks.js';
+import { useAuthUser } from '~/slices/auth/auth.js';
+import { uploadAvatar } from '~/slices/files/actions.js';
+
+import { type EditAvatarFormData } from './libs/types/types.js';
+import styles from './styles.module.scss';
+
+const AVATAR_SIZE = 200;
+
+const EditAvatarForm: React.FC = () => {
+  const { handleSubmit, setValue, register, formState } =
+    useAppForm<EditAvatarFormData>({
+      defaultValues: { avatar: undefined },
```

Comment:
it's better to use null for default values

Reply:
undefined is being added by the react-hook-form, because 
type EditAvatarFormData = { avatar: FileList };
If i make it
type EditAvatarFormData = { avatar: FileList | null };
the resulting type of the field would be
avatar: FileList | null | undefined